deployment/test_env/.env

```diff
@@ -1,16 +1,16 @@
-ANGOLA_UI_VERSION=1.5.1-SNAPSHOT
+ANGOLA_UI_VERSION=1.5.2-RC1
 
-OL_REQUISITION_VERSION=8.3.2
-OL_REFERENCEDATA_VERSION=15.2.1
-OL_AUTH_VERSION=4.3.1
-OL_NOTIFICATION_VERSION=4.3.1
-OL_FULFILLMENT_VERSION=9.0.0
-OL_STOCKMANAGEMENT_VERSION=5.1.3
-OL_REPORT_VERSION=1.1.4-SNAPSHOT
```

Comment:
My mistake, please leave this version as 1.1.4-SNAPSHOT because this report indicates the ANgola custom report service - not the core one.